src/internal/hooks/use-table-interaction-metrics/__tests__/use-table-interaction-metrics.test.tsx

```diff
@@ -142,6 +150,7 @@ describe('useTableInteractionMetrics', () => {
         })
       );
 
+      jest.runAllTimers();
```

Comment:
Since the event is emitted on a debounce and we are using fake timers here we need to run them before we can assert